src/Commands/EnvPushCommand.php

```diff
@@ -22,15 +22,19 @@ class EnvPushCommand extends BuildToolsBase
      *
      * @param string $site_env_id Site and environment to push to. May be any dev or multidev environment.
      * @param string $repositoryDir Code to push. Defaults to cwd.
+     * @option label What to name the environment in commit comments
```

Comment:
These two lines for `label` and `message` are not material to why I made this PR. But it seemed like a bug to be omitting them as command line options when they are present as options in the function signature.